.cirrus.yml

```
@@ -0,0 +1,22 @@
+aliases:
+  bundle_cache: &bundle_cache |
```

My bad. `|` is redundant. Cirrus CI is using SnakeYAML for parsing and this is the [format it supports](https://bitbucket.org/asomov/snakeyaml/wiki/Documentation#markdown-header-aliases). Maybe YAML standard allows several forms. 🤔 

```suggestion
  bundle_cache: &bundle_cache
```

---

I tried this, and no: https://cirrus-ci.com/build/5926221328154624

---

> Maybe YAML standard allows several forms. :thinking:

According to the official reference, no.

---

Found an issue in SnakeYML, implemented a workaround, should be deployed in ~20-30 minutes. Will ping you when it's available. Thanks for reporting and sorry for the inconvenience.

---

This should work now. You can try to re-trigger your build here: https://cirrus-ci.com/build/5926221328154624

---

Thank you! I glad to find the issue.